integration/tests-ssr/request-service.js

```diff
@@ -4,9 +4,10 @@ module.exports = async value =>
   new Promise((resolve, reject) => {
     request.get(value, { proxy: '' }, (error, response, data) => {
       if (error) {
-       console.log(error);
-        reject(error);
+        console.log(error);
+        return reject(error);
```

Comment:
What does it mean to return something from a promise callback?
If it is just to avoid the else then maybe this would be clearer:
```suggestion
        reject(error);
        return;
```